person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, parent_permission, name = 'Unknown')
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
```

Comment:
- [ ] Please make sure the `can_use_services?` is a public method instead of the private method, only the `of_age?` method should be private as requested in the project requirements.